agent/bench-scripts/pbench-uperf

```diff
@@ -24,12 +24,12 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 # source the base script
 . "$pbench_bin"/base
 
-benchmark_rpm=$script_name
 benchmark="uperf"
+benchmark_rpm=${benchmark}
 if [[ -z "$benchmark_bin" ]]; then
     benchmark_bin=/usr/local/bin/$benchmark
 fi
-ver=1.0.4
+ver=1.0.7
```

Comment:
We should probably do this in pbench-agent-default.cfg, the same way we do fio.

Reply:
Done.